BrainPortal/app/views/tool_configs/_form_fields.html.erb

```diff
@@ -112,6 +112,16 @@
     </p>
   </span>
 
+  <span title="Index of the container image">
+    <p><%= f.label :container_index_location, "Index of the container image:" %><br/>
+      <%= f.text_field :container_index_location %><br>
+      <div class="field_explanation">
+        The index/url of the Container image in which the docker/singularity container is
```

Comment:
`Container` is capitalized for no reason; add examples for singularity too (and show default for singularity)